blueocean-config/src/test/java/io/jenkins/blueocean/config/FeaturesTest.java

```diff
@@ -0,0 +1,21 @@
+package io.jenkins.blueocean.config;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class FeaturesTest {
```

Comment:
@vivek these tests currently fail because JUnit isn't picking up the `features.properties` in the test resources. Any ideas?